ios/Exponent/Versioned/Modules/Api/EXSpeech.h

```diff
@@ -0,0 +1,7 @@
+#import <AVFoundation/AVFoundation.h>
+#import <React/RCTEventEmitter.h>
+#import <React/RCTBridge.h>
+
+@interface EXSpeech : RCTEventEmitter <AVSpeechSynthesizerDelegate>
```

Comment:
I think we can move the `AV` import and the `<protocol>` into the implementation file, no particular need to expose this in the header.